### PR TITLE
i18n(fr) Update french translation with last PRs

### PR DESCRIPTION
**Pull request by McFlyPartages**
#### Description (required)

Update french translation for multiple file outdated

### CODE REVIEW

File: src/content/docs/fr/install/manual.mdx [hunk 1]
```diff
@@ -8,11 +8,11 @@ import FileTree from '~/components/FileTree.astro';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 import ReadMore from '~/components/ReadMore.astro'
 
-Ce guide vous montrera comment installer et configurer manuellement un nouveau projet Astro si vous préférez ne pas utiliser [l'outil de ligne de commande automatique](/fr/install/auto/).
+Ce guide vous montrera comment installer et configurer manuellement un nouveau projet Astro.
 
-##### Vous préférez une méthode plus rapide pour commencer?
-
-<Button href="/fr/install/auto/">Essayez l'assistant de CLI `create astro` →</Button>
+:::tip[Vous préférez une méthode plus rapide pour commencer?]
```
Comment (thomasbnt) [inline src/content/docs/fr/install/manual.mdx:13]:
```suggestion
:::tip[Vous préférez une méthode plus rapide pour commencer ?]
```

File: src/content/docs/fr/guides/integrations-guide/alpinejs.mdx [hunk 2]
```diff
@@ -90,37 +90,69 @@ export default defineConfig({
...
+}
+```
+
+## Utilisation
+
+Une fois l'intégration installée, vous pouvez utiliser les directives et la syntaxe [Alpine.js](https://alpinejs.dev/) dans n'importe quel composant Astro. Le script Alpine.js est automatiquement ajouté et activé sur chaque page de votre site web. Ajoutez les scripts du plugin à la page `<head>`.
 
-  // Optionnel: Étendre Alpine.js
-  // Alpine.directive('foo', ...)
+L'exemple suivant ajoute le plugin [Alpine's Collapse plugin](https://alpinejs.dev/plugins/collapse)  pour développer et réduire le texte d'un paragraphe.
```
Comment (thomasbnt) [inline src/content/docs/fr/guides/integrations-guide/alpinejs.mdx:124]:
```suggestion
L'exemple suivant ajoute le plugin [Alpine's Collapse](https://alpinejs.dev/plugins/collapse)  pour développer et réduire le texte d'un paragraphe.
```
ça évite la répétition du mot "plugin"

**Review by thomasbnt — CHANGES_REQUESTED** [review]
Two suggestions about space in a note and a duplication of the word "plugin"

**Comment by McFlyPartages** [comment]
> Two suggestions about space in a note and a duplication of the word "plugin"

Thanks it's ok now

**Review by thomasbnt — APPROVED** [review]
LGTM!

**Review by VoxelMC — APPROVED** [review]
Lots of work! Thank you very much @McFlyPartages :)

I added 3 suggestions to remove the `DontEditWarning` component, as the import is not needed in the translated versions. 

Awesome work!!!

`As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

**Comment by McFlyPartages** [comment]
> Lots of work! Thank you very much @McFlyPartages :)
> 
> I added 3 suggestions to remove the `DontEditWarning` component, as the import is not needed in the translated versions.
> 
> Awesome work!!!
> 
> `As a non-native speaker, I reviewed this PR using the Docs Translation Review Checklist and it LGTM!`

Thanks

**Review by dreyfus92 — APPROVED** [review]
LGTM, merci à tous 😁

**Pull request merged**